charts/hivemq-swarm/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.monitoring.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    release: {{ .Release.Name }}
+    {{- include "hivemq-swarm.labels" . | nindent 4 }}
+  name: {{ include "hivemq-swarm.fullname" . }}
+  {{- if .Values.monitoring.serviceMonitor.namespace }}
+  namespace: {{ .Values.monitoring.serviceMonitor.namespace }}
+  {{- end }}
+spec:
+  endpoints:
+    - interval: {{ .Values.monitoring.serviceMonitor.interval }}
```

Comment:
so this defines the targets for both commander and API? does this lead to both targets being registered in prometheus?

Reply:
Yes - it will lookup both endpoints and since commander/agent have different container ports - they will be found individually.
Example 1 commander, 2 agents

![image](https://user-images.githubusercontent.com/6403243/112154085-37ce3f80-8be4-11eb-9bf1-973c368137d2.png)